src/ServerProvider.php

```diff
@@ -76,6 +98,15 @@ public function getRandomServerFromCluster(string $cluster): Server
         return $cluster->getServerByHostname($randomServerIndex);
     }
 
+    public function getRandomServerFromClusterByTag(string $cluster, string $tag): Server
+    {
+        $cluster = $this->getCluster($cluster);
+
+        $randomServerIndex = array_rand($cluster->getServersByTag($tag), 1);
+
+        return $cluster->getServerByHostname($randomServerIndex);
```

Comment:
Not seems that `$randomServerIndex` contains index. Technically thats maybe right, but semantically that variable store `hostname`.

Reply:
Changed variable name to ```$randomServerHostname```.